src/reference/asciidoc/amqp.adoc

```diff
@@ -3764,6 +3764,23 @@ The timers are named `spring.rabbitmq.listener` and have the following tags:
 
 You can add additional tags using the `micrometerTags` container property.
 
+Also see <<observation>>.
+
+[[observation]]
+===== Micrometer Observation
+
+Using Micrometer for observation is now supported, since version 3.0, for the `RabbitTemplate` and listener containers.
+
+Set `observationEnabled` on each component to enable observation; this will disable <<micrometer,Micrometer Timers>> because the timers will now be managed with each observation.
+
+Refer to https://micrometer.io/docs/tracing#_running_integration_tests[Micrometer Tracing] for more information.
```

Comment:
Why do you point here to the testing docs?
Why just not use a root link instead - https://micrometer.io/docs/tracing ?
```suggestion
Refer to https://micrometer.io/docs/tracing[Micrometer Tracing] for more information.
```

Reply:
Oops - had two pages open and copied the wrong link 😄